client/lang/en.js

```diff
@@ -23,7 +23,8 @@ if (typeof(ss) === 'undefined' || typeof(ss.i18n) === 'undefined') {
     "LinkField.LINK_DRAFT_TITLE": "Link has draft changes",
     "LinkField.LINK_MODIFIED_LABEL": "Modified",
     "LinkField.LINK_MODIFIED_TITLE": "Link has unpublished changes",
-    "LinkField.SAVE_RECORD_FIRST": "Cannot add links until the record has been saved",
+    "LinkField.SAVE_RECORD_FIRST": "Cannot add link until the record has been saved",
+    "LinkField.SAVE_RECORD_FIRST_MULTI": "Cannot add links until the record has been saved",
```

Comment:
We should avoid changing existing localisation keys because we don't use version control in transifex, so this could change the localisation key from _previous_ versions of linkfield as well if their localisations are updated.

It might also trip anyone up who's using this key for their own purposes.

Instead, just keep using the old key for the text that isn't changing and only add a new one.
```suggestion
    "LinkField.SAVE_RECORD_FIRST": "Cannot add links until the record has been saved",
    "LinkField.SAVE_RECORD_FIRST_SINGLE": "Cannot add link until the record has been saved",
```
You'll need to update the JSON and LinkField.js as well.

Reply:
Ah dang.

Makes sense there.

Too big of change...

I suspect Transifex would have some sort of auto invalidate certain things mechanic but yeah, less work this way.